e2e/src/test/java/no/sysco/testing/kafka/e2e/DockerComposeIT.java

```diff
@@ -0,0 +1,77 @@
+package no.sysco.testing.kafka.e2e;
+
+import io.restassured.RestAssured;
+import io.restassured.http.ContentType;
+import java.io.File;
+import java.util.Arrays;
+import java.util.List;
+import java.util.UUID;
+import java.util.concurrent.TimeUnit;
+import no.sysco.testing.kafka.e2e.representation.MessageJsonRepresentation;
+import org.junit.ClassRule;
+import org.junit.Test;
+import org.testcontainers.containers.DockerComposeContainer;
+import org.testcontainers.containers.wait.strategy.Wait;
+
+import static org.awaitility.Awaitility.await;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+public class DockerComposeIT {
+
+    public static final String JSON_SERVER_EXPOSED = "http://localhost:3000";
+    public static final String HTTP_PRODUCER_EXPOSED = "http://localhost:8080";
```

Comment:
nit: I'd rephrase these 2 to something like: `HTTP_PRODUCER_BASE_URL`

Reply:
done